Source/WebKit/Shared/WebCoreArgumentCoders.serialization.in

```diff
@@ -7749,3 +7749,10 @@ class WebCore::SerializedPlatformDataCueValue {
     std::variant<std::nullptr_t, RetainPtr<NSString>, RetainPtr<NSDate>, RetainPtr<NSNumber>, RetainPtr<NSData>> value;
 #endif
 };
+
+#if PLATFORM(COCOA)
+header: <WebCore/SharedBuffer.h>
+[RefCounted, CustomHeader] class WebCore::FragmentedSharedBuffer {
+    std::optional<WebCore::SharedMemoryHandle> toSharedMemoryHandle();
```

Comment:
Now we're creating a shared memory even if `bufferSize < minimumPageSize`, which seems like a performance regression compared to the previous code.